umh-core/pkg/fsmv2/cpu/cpu_admission_deadline_warning_test.go

```
@@ -0,0 +1,310 @@
+const testHierarchyPath = "scenario-test(application)/cpu-test(" + WorkerType + ")"
+
+// newDepsWithLogger is newDeps with an explicit logger, so a test can observe
+// SentryWarn calls (newDeps always binds a Nop logger).
+func newDepsWithLogger(log deps.FSMLogger, s cpuhealth.Sampler, cores, quota float64) *CPUDeps {
+	d := newDeps(s, cores, quota)
+	d.BaseDependencies = deps.NewBaseDependencies(log, nil, deps.Identity{
+		ID:            "cpu-test",
+		WorkerType:    WorkerType,
+		HierarchyPath: testHierarchyPath,
+	})
+	return d
```

golangci-lint (2.12.2)

[error] return with no blank line before

(nlreturn)